generators/pythonv2/codegen/src/ast/Annotation.ts

```diff
@@ -0,0 +1,27 @@
+import { AstNode } from "./core/AstNode";
+import { Writer } from "./core/Writer";
+
+export declare namespace Annotation {
+    interface Args {
+        /* The type hint */
+        type: string | AstNode;
+    }
+}
+
+export class Annotation extends AstNode {
+    private type: string | AstNode;
+
+    constructor(args: Annotation.Args) {
+        super();
+        this.type = args.type;
+    }
+
+    public write(writer: Writer): void {
+        writer.write(": ");
+        if (typeof this.type === "string") {
+            writer.write(this.type);
+        } else {
+            this.type.write(writer);
```

Comment:
[q] How does the writer handle super long lines? like let's say the AstNode is a `Union` of like 30 types

Reply:
The csharp package has a formatting step: https://github.com/fern-api/fern/blob/main/generators/csharp/codegen/src/ast/core/AstNode.ts#L85

Maybe we do something similar with `black`? I'd be keen on @dsinghvi's advice here when the time comes.

Reply:
I would recommend using this wasm library that uses ruff https://www.npmjs.com/package/@wasm-fmt/ruff_fmt